spec/services/client_login_service_spec.rb

```diff
@@ -97,14 +97,6 @@
           it "returns a blank set" do
             expect(subject.clients_for_token("raw_token")).to match_array []
           end
-
```

Comment:
I can't say I precisely understand right now the purpose of this test block that got deleted. What was it doing, and why is it safe to delete? I could help figure out what it was for, if that's useful.

Reply:
there used to be unique behavior for drop off clients. the behavior is now the same for clients/intakes of all types, so it does not make sense to maintain tests that test the unique logic of a certain type of return.